.github/workflows/ci.yml

```diff
@@ -3,9 +3,9 @@ name: Continuous Integration
 # On every pull request on the develop branch, execute all tests [CI]
 on:
   pull_request:
```

Comment:
Think this looks good - I'm not super familiar with GitHub actions, but would it be simple to update this to also run all tests on every commit, as well as on each pull request? Most times I've worked with CI in the past it's usually been on each commit.

Reply:
I will have a look.